webui/react/src/stores/cluster.tsx

```
@@ -240,6 +243,23 @@ class ClusterStore extends PollingStore {
     return () => canceler.abort();
   }
 
+  public fetchKubernetesResourceManagers(signal?: AbortSignal): () => void {
+    const canceler = new AbortController();
+    getKubernetesResourceManagers(undefined, { signal: signal ?? canceler.signal })
+      .then((response) => {
+        this.#kubernetesResourceManagers.set(Loaded(response.names));
+      })
+      .catch((e) => {
+        handleError(e, {
+          level: ErrorLevel.Error,
+          publicMessage: 'Failed to fetch Resource Managers.',
+          silent: true,
```

ditto re: silent errors